crypto-core/src/main/java/com/palantir/crypto2/io/CryptoStreamFactory.java

```diff
@@ -44,14 +46,25 @@
 
     private static volatile boolean fullExceptionLoggedAlready = false;
```

Comment:
I wrote this as an entirely additive change. We could get rid of the `fullExceptionLoggedAlready` flag, and the catch+fallback case entirely, but there may be cases that helps us which I'm not aware of, so I opted to minimize risk.

Reply:
I'm good with killing consolidating the exception handling. There aren't any other cases I'm aware of where cipher construction can throw so can also probably remove the try catch.